managing_providers/_topics/adding_a_vmware_vcenter_provider.md

```diff
@@ -25,7 +25,9 @@ environment, add a VMware vCenter provider to the appliance.
 6.  Select the appropriate **Zone** for the provider. By default, the
     zone is set to **default**.
 
-7.  In the **Credentials** area, under **Default**, provide the login
+7.  Optional: Enter the **Host Default VNC Start Port** and **Host Default VNC End Port**. These properties are used for remote console access with VNC. If VNC is available from the hosts, list the available port range using the start and end port. **Note**: This is only available for VMware ESXi hosts.     
```

Comment:
> **Note**: This is only available for VMware ESXi hosts.

This is redundant when we're in the "Add a vCenter" section since ESXi are the only type of host that are available on vCenter.

Reply:
removed the **Note**: This is only available for VMware ESXi hosts.